integration/cypress/integration/login/login.spec.ts

```diff
@@ -48,22 +49,18 @@ context("Login page", () => {
   });
 
   it("logs in and redirects to the user intro if setup intro complete", () => {
-    // Log in - should go to setup intro
+    // Log in - should go to setup intro.
     cy.get("input[name='username']").type(Cypress.env("username"));
     cy.get("input[name='password']").type(Cypress.env("password"));
     cy.get("button[type='submit']").click();
     cy.location("pathname").should("eq", generateNewURL("/intro"));
 
-    // Open the skip confirmation.
-    cy.get("button[data-test='secondary-submit']").click();
-
-    // Confirm skipping setup intro - should redirect to user intro.
-    cy.get("button[data-test='action-confirm']").click();
-    cy.location("pathname").should("eq", generateNewURL("/intro/user"));
-
     // Log out.
     cy.get(".p-navigation__link a:contains(Log out)").click();
 
+    // Set cookie to skip setup intro.
+    cy.setCookie("skipsetupintro", "true");
+
```

Comment:
I've replaced this bit with just setting the cookie to keep state between tests as consistent as possible (maybe a fool's errand)